### ISSUE TITLE
The Seniorious can no longer be found in the new version

### ISSUE THREAD

**Issue by boombbo** [Carzit/sd-webui-samplers-scheduler#8]
I don't know if this project is still under maintenance, Just found that the webUI was in the process of being updated, Decouple the sampler and scheduler, It is no longer possible to find it in the sampler Seniorious, I'm looking forward to seeing it in comfyUI, I'm also trying to get a little more tricky to try custom samplers

**Comment by Carzit** [comment]
Yes, note that SD WebUI has implemented the separation of samplers and noise schedules since version 1.9.0. However, this extension was developed based on versions 1.6 and 1.7.   

I am sorry that due to lack of attention, maintenance was not carried out in time.I will plan to adapt to the 1.9.x version later.   

As for comfyUI, forgive me for not having much contact with it, but if you have any good ideas or information worth sharing, please let me know.

**Comment by Carzit** [comment]
In fact, it was more out of a concern for safety than anything else. The seniorious extension actually will not load in SD WebUI version higher than 1.6.x (see scripts/load_Seniorious.py); I'll make appropriate changes after confirming compatibility. 

Again, thank you for your issue submission, which reminds me that I - a college student who is bogged down with endless chores and useless classes - still have valuable things that should be cherished and maintained :)
